docs/features.md

```diff
@@ -0,0 +1,64 @@
+# Features
+
+This document gives an overview over Alacritty's features beyond its terminal
+emulation capabilities. To get a list with supported control sequences take a
+look at [Alacritty's escape sequence support](./escape_support.md).
+
+## Vi Mode
+
+The vi mode allows moving around Alacritty's viewport and scrollback using the
+keyboard. It also serves as a jump-off point for other features like search and
+opening URLs with the keyboard. By default you can launch it using
+<kbd>Ctrl</kbd> <kbd>Shift</kbd> <kbd>Space</kbd>.
+
+### Motion
+
+The cursor motions are setup by default to mimic vi, however they are fully
+configurable. If you don't like vi's bindings, take a look at the [configuration
+file] to change the various movements.
+
+### Selection
+
+One useful feature of vi mode is the ability to make selections and copy text to
+the clipboard. By default you can start a selection using <kbd>v</kbd> and copy
+it using <kbd>y</kbd>. All selection modes that are available with the mouse can
+be accessed from vi mode, including the semantic (<kbd>Alt</kbd> <kbd>v</kbd>),
+line (<kbd>Shift</kbd> <kbd>v</kbd>) and block selection (<kbd>Ctrl</kbd>
+<kbd>v</kbd>). You can also toggle between them while the selection is still
+active.
+
+### Opening URLs
+
+While in vi mode you can open URLs using the <kbd>Enter</kbd> key. If some text
+is recognized as a URL, it will be underlined once you move the vi cursor above
+it. The program used to open these URLs can be changed in the [configuration
+file].
+
+## Search
+
+Search allows you to find anything in Alacritty's scrollback buffer, it can be
+launched either directly (<kbd>Ctrl</kbd> <kbd>Shift</kbd> <kbd>f</kbd>), or
+from vi mode (<kbd>/</kbd>).
+
+### Vi Search
+
+When using search while the vi mode is active, it can be used to quickly move
+around the scrollback buffer. The bindings <kbd>Ctrl</kbd> <kbd>n</kbd> and
+<kbd>Ctrl</kbd> <kbd>Shift</kbd> <kbd>n</kbd> allow you to navigate within
```

Comment:
Those are just `n` and `shift + n`, no ctrl is involved here. Also, `plus`.

Reply:
That is incorrect, I'm explicitly talking about motions **within** the search here.

Reply:
I mean we don't have `Ctrl + Shift + N` or `Ctrl + n` in a alacritty.yml or implicitly in terminal, I'm not sure what is incorrect here.

Reply:
Oh right, these aren't bound to anything by default. I completely forgot about that.

Do you think it would make sense to bind these by default? Do you have them bound to something?

Reply:
>Do you think it would make sense to bind these by default? Do you have them bound to something?

I don't think that we should.

Reply:
This is still not addressed. Just removing the sentence about `ctrl + shift + n` should solve it.

Reply:
Should we then document that these bindings can be used to jump within matches? I think it's an important detail of vi search.

Reply:
>Should we then document that these bindings can be used to jump within matches? I think it's an important detail of vi search.

You mean `Actions`? we don't have the bindings you're talking about.

Reply:
No, I mean that people can create these bindings to jump around within matches.

Reply:
people can create whatever bindings they want. They could bind certain actions to have an ability to move within the search, however actions that we have to navigate around, are not bound to a 'current' match, they just traverse the matches in a defined order. So I do feel like there's no value in that sentence at all.

Reply:
How is there no value in documenting that a binding exists that is not bound by default which would likely be very useful to a lot of people?

Reply:
Well, we have other bindings not bound by default, which also has a value to begin with.

>How is there no value in documenting that an action exists that is not bound by default which would likely be very useful to a lot of people

You can say that there're other ways to move Vi cursor other than default `n` and `N` ans just list the actions folks can use, describing what they should do. However if you want to describe a key binding that may be of use maybe use config's key binding section? For me features file is more like, 'things you can try right away without touching config at all'.

Reply:
Using the configuration file's key binding section would likely make this harder to discover though.

Reply:
I've reworded things a little bit, let me know what you think.